lib/fontdetect.js

```diff
@@ -101,6 +108,7 @@ FontDetect = function()
             // Our hashtable of optional params.
             var msInterval = (p_options && p_options.msInterval) ? p_options.msInterval : 100;
             var msTimeout  = (p_options && p_options.msTimeout) ? p_options.msTimeout : 2000;
+			eLang = (p_options && p_options.lang && p_options.lang in aChars) ? p_options.lang : 'en';

```

Comment:
p_options.lang needs to be changed as well.

eLang = (p_options && p_options.eLang && p_options.eLang in aChars) ? p_options.eLang : 'en';